include/yaclib/util/result.hpp

```diff
@@ -127,13 +143,29 @@ class Result {
       return std::get<V>(std::move(_result));
     }
   }
+
+  [[nodiscard]] CrefT<V> Value() & noexcept {
```

Comment:
```suggestion
  [[nodiscard]] ConstValue Value() const& noexcept {
```
Also plz, use MoveValue for `Value() &&`